tests/baselines/reference/api/typescript.d.ts

```diff
@@ -1742,12 +1743,20 @@ declare namespace ts {
         readonly kind: SyntaxKind.JSDocComment;
         readonly parent: HasJSDoc;
         readonly tags?: NodeArray<JSDocTag>;
-        readonly comment?: string;
+        readonly comment?: JSDocComment;
```

Comment:
Will this need to be marked as an [API breaking change](https://github.com/microsoft/TypeScript/wiki/API-Breaking-Changes)? If so, can you document it?

Reply:
Yes, where should I do that?